lib/home/home_screen.dart

```diff
@@ -39,7 +39,7 @@ class _MyHomeScreenState extends State<MyHomeScreen> {
 
   final List<Widget> _screens = [
     const ShoppingListScreen(),
-    const SettingsScreen(),
+    const SettingsWidget(),
```

Comment:
```suggestion
    const SettingsScreen(),
```
Estamos seguindo um padrão nas view adicionando o sufixo `Screen`. Se achar que podemos manter esse padrão.